src/status_im2/contexts/quo_preview/markdown/text.cljs

```diff
@@ -31,24 +30,14 @@
               {:key   :monospace
                :value "Monospace"}]}])
 
-(defn cool-preview
+(defn preview-text
```

Comment:
Just `view` would be enough for all preview namespaces, since each namespace only expose one component var. If you consider this to be out of scope it's fine :) We can later on create a simple PR that does just this.

Reply:
That's right view or preview is enough, but it's a bit out of scope :)